ui.go

```diff
@@ -441,49 +441,43 @@ func (win *win) printDir(screen tcell.Screen, dir *dir, context *dirContext, dir
 			}
 		}
 
+		// make space for select marker, and leave another space at the end
+		maxWidth := win.w - lnwidth - 2
+		// make extra space to separate windows if drawbox is not enabled
+		if !gOpts.drawbox {
+			maxWidth -= 1
+		}
+
 		var s []rune
 
+		// leave space for displaying the tag
```

Comment:
(No action needed) I would prefer treating the selection marker and the tag marker the same way in the code, either by appending two spaces here or by changing the column where the next `print` statement happens.

Either way, the `maxWidth` above would need to be adjusted.

One of us can fix this in a future PR.

Reply:
The difference between the selection and tag markers is that the tag marker follows the style of the `cursor...fmt` options whereas the formatting of the selection marker is completely indepedent. I think this is supposed to be intended from the comments in #791 when tags were originally added.

Again I'm uninclined to make any changes to the UI, since the topic is subjective and based on personal preferences. I'm worried a situation similar to #1258 will happen, although that wasn't so bad because it could still be solved by configuration.

Reply:
This is meant to be a no-op change.

I think you did catch onto a problem in my reasoning: only one of the two alternatives I gave ("changing the column where the next print statement happens") is viable. The other option I gave would indeed print the selection marker in the wrong color.

Reply:
> The difference between the selection and tag markers is that the tag marker follows the style of the cursor...fmt options whereas the formatting of the selection marker is completely indepedent

Ah, I'm beginning to understand what you said here. I think you're right, neither of my approaches look like no-ops. I'll think about it some more.

Reply:
Feel free to mark this conversation as resolved as well.

Reply:
Done, thanks for having a look at the changes.